doctrine.rst

```diff
@@ -523,10 +523,10 @@ be able to go to ``/product/1`` to see your new product::
         // return $this->render('product/show.html.twig', ['product' => $product]);
     }
     
-This in an alternative example using Symfony's autowiring and Dependencyinjection : 
-    
+Another possibility is to use the ``ProductRepository`` injected by the dependency injection container::
```

Comment:
autowiring should explicitly mentioned here as this is responsible, not the DIC standalone, to use it this way

Reply:
done